make-release.sh

```diff
@@ -82,9 +83,7 @@ bump_version() {
   git checkout ${BUMP_BRANCH}
 
   echo "[INFO] Update project version to ${NEXTVERSION}"
-  updateYaml ${NEXTVERSION}
-
-  if [[ ${docommit} -eq 1 ]]; then
+  if updateYaml ${NEXTVERSION} && [[ ${docommit} -eq 1 ]]; then
```

Comment:
so now none of the commit/push/pull/PR will happen if there's nothing changed in the yaml. Nice improvement!